modin/pandas/test/test_groupby.py

```diff
@@ -1361,7 +1363,9 @@ def test_groupby_multiindex(groupby_kwargs):
     )
     modin_groupby_equals_pandas(md_grp, pd_grp)
     df_equals(md_grp.sum(), pd_grp.sum())
-    df_equals(md_grp.size(), pd_grp.size())
+    if PandasCompatVersion.CURRENT != PandasCompatVersion.LATEST:
+        # due to https://github.com/pandas-dev/pandas/issues/48738, that would fail on pandas 1.5.0
+        df_equals(md_grp.size(), pd_grp.size())
```

Comment:
Should we fix this as part of a separate issue?

Reply:
Yes, I was planning on making a tracker for that, but haven't made one yet. Will do.

Reply:
Please put it here once created.

Reply:
replaced pandas tracker with Modin one